UnityProject/Assets/Scripts/Clothing/BackPack/InteractableStorage.cs

```diff
@@ -167,6 +178,12 @@ public bool WillInteract(HandApply interaction, NetworkSide side)
 		if (allowedToInteract == false) return false;
 		// Use default interaction checks
 		if (DefaultWillInteract.Default(interaction, side) == false) return false;
+		if (itemStorage.GetItemSlots().All(slot => slot.IsOccupied) && interaction.HandObject != null)
+		{
+			Chat.AddExamineMsg(interaction.Performer,
+				$"<color=red>The {interaction.HandObject.ExpensiveName()} won't fit in the {interaction.HandObject.ExpensiveName()}, Make some space!</color>");
```

Comment:
```suggestion
				$"<color=red>The {interaction.HandObject.ExpensiveName()} won't fit in the {interaction.HandObject.ExpensiveName()}. Make some space!</color>");
```
or
```suggestion
				$"<color=red>The {interaction.HandObject.ExpensiveName()} won't fit in the {interaction.HandObject.ExpensiveName()}, make some space!</color>");
```

Reply:
![image](https://user-images.githubusercontent.com/34368774/142770303-c2797613-eb86-4c10-9da2-b96209b3938b.png)
Also if we put a `.` then the word after it must be capitalized.